### ISSUE TITLE
Windows 10 taskbar on top of the game

### ISSUE THREAD

**Issue by KleoPetroff** [SamHurne/gw2pao#48]
After the last Windows 10 update, the taskbar started to appear on top of the game. If GW2P is stopped, the taskbar is properly hidden. Is there something I do on my side to fix the problem?

![](http://i.imgur.com/yBrhvx0.png)

**Comment by joaroyen** [comment]
I noticed that Sam has already fixed this in the develop branch. I built that branch myself locally and can confirm that the taskbar is no longer on top of GW2. If you have Visual Studio on your machine you can try to create a build yourself, otherwise you need to wait for the next official release.

**Comment by SamHurne** [comment]
Yes, this is already "fixed" on the develop branch. I don't understand the exact cause yet, but something in one of the new windows 10 updates changed the taskbar's behavior so that it always appears unless the WPF "ShowInTaskbar" window property is set to false.

I know it's a rather frustrating issue. No promises, but I'm going to see about getting a new release out sooner than later.

**Comment by KleoPetroff** [comment]
After building the develop branch, I can confirm that the taskbar is no longer over the game.

**Comment by itissean** [comment]
Also confirmed that the taskbar no longer appears while in windowed fullscreen on develop branch.

**Comment by ecspresso** [comment]
Is it possible to add this fix to the current version?

**Comment by Dajova** [comment]
Can confirm that its still being a issue, this is the only reason why i haven't 100% gone over to this app instead of using Overwolf. :/

![Image](https://i.imgur.com/mLsPfWi.jpg)
